src/azure-cli/azure/cli/command_modules/vm/commands.py

```diff
@@ -340,9 +340,9 @@ def load_command_table(self, _):
         g.custom_command('list', 'list_vm_extension_images')
 
     with self.command_group('vm image', compute_vm_image_sdk) as g:
-        g.command('list-offers', 'list_offers')
-        g.command('list-publishers', 'list_publishers')
-        g.command('list-skus', 'list_skus')
+        g.custom_command('list-offers', 'list_offers')
+        g.custom_command('list-publishers', 'list_publishers')
```

Comment:
Should the method name be consistent with other custom methods without `_edge_zone`?

Reply:
`Edge_zone` is just one of the cases, is it better not to add？

Reply:
Yes, exactly